### DISCUSSION TITLE
How override cells padding?

### DISCUSSION THREAD

**Discussion by iakovalgaev** [table-library/react-table-library#38] [Q&A] [answered]
When I try to change the padding. The width and min-width for specified in useTheme are redefined.

![image](https://user-images.githubusercontent.com/61377069/161733194-f6534984-b196-40da-b7ad-f2667d0abd8c.png)
![image](https://user-images.githubusercontent.com/61377069/161734270-d8503e2e-968f-4228-a1c2-f6bdbd26d8d6.png)

**Comment by rwieruch** [comment, marked as answer]
Here you can see an example how to change the padding of the Cell components: https://react-table-library.com/?path=/docs/features-theme--dense
